Bump xdg-home to 1.1 and explicitly enable "user" feature in nix.
It seems that the feature was always missing, but previous
versions of xdg-home masked this since they used the user feature
in nix.

Index: zbus/Cargo.toml
===================================================================
--- zbus.orig/Cargo.toml
+++ zbus/Cargo.toml
@@ -155,7 +155,7 @@ version = "0.2"
 optional = true
 
 [dependencies.xdg-home]
-version = "1.0.0"
+version = "1.1.0"
 
 [dependencies.zbus_macros]
 version = "=3.14.1"
@@ -237,5 +237,6 @@ version = "0.26.0"
 features = [
     "socket",
     "uio",
+    "user",
 ]
 default-features = false
